cudapoa/src/cudapoa_nw.cuh

```diff
@@ -472,22 +474,22 @@ __global__ void runNeedlemanWunschKernel(uint8_t* nodes,
                                          int32_t match_score,
                                          SizeT* aligned_nodes)
 {
```

Comment:
here I copy a summary of my discussion with Andreas on slack:
removing `SizeT` template parameter results in linking error of multiple definitions found. Adding `inline` fixes the problem, but adds multiple instances of this `warning: inline qualifier ignored for "__global__"`. Another option is to use `static` instead of `inline`, where it creates a separate copy of the kernel per object file. 
To avoid compiler warnings, for now the kernel is kept templetized and `static_assert` is added to verify `SizeT` is `int16_t`

Reply:
done.